test/utils/node-test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * A port of tap's `t.type` that can be used with `node:assert`
+ * https://github.com/tapjs/tapjs/blob/511019b2ac0fa014370154c3a341a0e632f50b19/src/asserts/src/index.ts#L199
+ */
```

Comment:
Please include the license of that file if copied over.

Reply:
Thank you, [88d2c96](https://github.com/nodejs/undici/pull/2564/commits/88d2c96c101ae83455a94752b93f5eabd59aeaaa)